### ISSUE TITLE
XDPM is much better then DDIM(default)

### ISSUE THREAD

**Issue by ptits** [kijai/ComfyUI-CogVideoXWrapper#270]
kijai, thanks a lot, it works like a blast from the first try (ubuntu A100)

I got MUCH better results with XDPM then with DDIM

found this in their docs:
    # We recommend using `CogVideoXDDIMScheduler` for CogVideoX-2B.
    # using `CogVideoXDPMScheduler` for CogVideoX-5B / CogVideoX-5B-I2V.

indeed, see below

https://github.com/user-attachments/assets/4a14e250-9e64-4dbc-a6ea-774407560d18

https://github.com/user-attachments/assets/edd2eeae-550b-4404-a27d-b25e85b8bc51

**Comment by kijai** [comment]
Technically it is better yes, but in reality I find it varies on the task. Different schedulers should absolutely be tested and my defaults aren't really meant to be universal defaults. Thanks for sharing this comparison!

**Comment by DmytroSokhach** [comment]
@ptits hey, do you notice that "speed" of animation is different between DDIM and XDPM?
I find that XDPM(default) is more natural on 24fps, and DDIM is like 2-3 times faster.

**Comment by DmytroSokhach** [comment]
@kijai I see inside CogVideo sources https://github.com/THUDM/CogVideo/blob/d82922cc799c24f5d0838afff9b672a102482697/inference/cli_demo.py#L123
`use_dynamic_cfg=True,  # This id used for DPM scheduler, for DDIM scheduler, it should be False`

My tests with DDIM are kind of not worthy. Is this `user_dynamic` property set automatically? Or is there plans to add it?

**Comment by kijai** [comment]
> @kijai I see inside CogVideo sources https://github.com/THUDM/CogVideo/blob/d82922cc799c24f5d0838afff9b672a102482697/inference/cli_demo.py#L123 `use_dynamic_cfg=True, # This id used for DPM scheduler, for DDIM scheduler, it should be False`
> 
> My tests with DDIM are kind of not worthy. Is this `user_dynamic` property set automatically? Or is there plans to add it?

Yes, what dynamic_cfg does is done automatically in my code when CogVideoXDPM is the scheduler.
